lib/pg.ddl.js

```diff
@@ -0,0 +1,323 @@
+'use strict';
+
+const { iter } = require('metarhia-common');
+
+const RESERVED = require('./sql.reserved.js');
+
+const SEPARATOR = '\n\n' + '-'.repeat(80) + '\n\n';
+
+const PREDEFINED_DOMAINS = {
+  Time: 'TIME WITH TIME ZONE',
```

Comment:
The PostgreSQL types for predefined domains are capitalized while the types in `typeFromDomain` are not. I think you should decapitalize the type names here.